_test/test_sqw/test_PixelData.m

```diff
@@ -69,8 +63,9 @@
     end
 
     function delete(obj)
-        rmpath(fullfile(obj.this_dir, 'utils'));
-        set(hor_config, obj.old_config);
+        if ismember(fullfile(obj.this_dir, 'utils'), split(path, {':', ';'}))
```

Comment:
should be 
`split(path,pathsep)`
`pathsep` -- system dependent  path separator

Reply:
hor_config 
needs description of page_size after     
%   mem_chunk_size    - Maximum length of buffer array to accumulate pixels
%                       from an input file.
something like:
% page_size  -- property to define page size in experimental file based mode